contribute/backend/communication.md

```diff
@@ -116,11 +112,11 @@ func (s *MyService) UserCreated(event *events.UserCreated) error {
 }
 ```
 
-**Tip:** Browse the available events in the `events` package.
+> **Tip:** To learn about the available events, refer to the documentation in the `events` package.
```

Comment:
for this one there's a tip shortcode: {{< admonition type="tip" >}}
This also applies to headings that contain a forward slash or parentheses or square brackets.
{{< /admonition >}}